_inc/client/plans/single-product-backup.jsx

```diff
@@ -6,6 +6,35 @@ import { translate as __ } from 'i18n-calypso';
 
 import './single-product-backup.scss';
 
+export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
+	const perYearPriceRange = `${ currencySymbol }${ backupPlanPrices.jetpack_backup_daily.yearly }-${ backupPlanPrices.jetpack_backup_realtime.yearly } /year`;
```

Comment:
This line is very long and har to read. What do you think of assigning the daily yearly and realtime yearly prices to some clearly-named constants first and then use them here? We could do the same with the monthly prices used below.

Also, there's a hardcoded "year" string inside. We need to make it available for translation.

Reply:
In addition to translating `/year`, I think we'd also need to be able to localize the whole string. In some countries the order of the words may be "/year - price - currency" for example.

Reply:
Agreed for both issues.

Reply:
Do you think you could create issues, or one issue to keep track of the things that need to be fixed before launch, like the above?

Reply:
> Do you think you could run `yarn build` in the environment where you commit, to avoid these errors and benefit from the things our pre-commit hook brings us, such as linting and prettyfying on commit?

I do this. For this case, because I split out each component into its own PR, the components were unused which gives an error that prevented me from committing. I only committed when this was the only error I got.

Reply:
I will do that, but in this case the issue is fixed in this PR.